js/toggleMenu.js

```diff
@@ -0,0 +1,26 @@
+const menu = document.querySelector('.menu');
+const menuItems = document.querySelectorAll('.menuItem');
+const hamburger = document.querySelector('.hamburger');
+const closeIcon = document.querySelector('.closeIcon');
+const menuIcon = document.querySelector('.menuIcon');
```

Comment:
- [ ] Kindly ensure that your application contains an hamburger menu on the left side and not at right to stick as much as possible Behance template, and ensure that you improve its design

![Screenshot from 2023-02-14 01-50-20](https://user-images.githubusercontent.com/83042742/218600461-b06e9210-19d0-4abc-955a-d36aeb311fe7.png)